README.md

```diff
@@ -42,44 +42,38 @@ To install depencidencies for development, replace the second command with:
 cd toppra && pip install -e .[dev]
 ```
 
-## C++ API
+## New C++ API
 
 `toppra` is also implemented as a stand-alone C++ library with minimal
 dependency. See `cpp/README.md` for installation instruction and the
 test cases for example usage. Be warned that this implementation is
 relatively new and is not yet stable enough for production use.
 
-# Development roadmap
-
-The following is a non-exhautive list of features that we are
-considering to include in the library.
-
-- Improve the trajectory / geometric path interface [#81](https://github.com/hungpham2511/toppra/issues/81)
-- Implement a C++ interface [#43](https://github.com/hungpham2511/toppra/issues/43)
-- Implement a C++ interface to popular motion planning libraries.
-- Improve the numerical stability of the solvers for degenerate cases.
-- Post-processing of output trajectories: [#56](https://github.com/hungpham2511/toppra/issues/56), [#80](https://github.com/hungpham2511/toppra/issues/80)
-
 # Supports
 
 ## Bug tracking
 Please report any issues, questions or feature request via 
 [Github issues tracker](https://github.com/hungpham2511/toppra/issues).
 
-Please provide more details on the errors/bugs that you encounter. The
-best way is to provide a Minimal Working Example that produces the
-reported bug and attach it with the issue report.
-
 ## Contributions
-Pull Requests are welcomed!
-- Go ahead and create a Pull Request and we will review your proposal!
-- For new features, or bug fixes, preferably the request should
-  contain unit tests. Note that `toppra` uses
-  [pytest](https://docs.pytest.org/en/latest/contents.html) for all
-  tests. Check out the test folder for more details.
+Pull Requests are welcomed! Go ahead to create a Pull Request and we
+will review your proposal!
+
+# Credits
+
+`toppra` was originally developed by [Hung
+PHam](https://hungpham2511.github.com/) (Eureka Robotics, former CRI
+Group) and [Pham Quang-Cuong](https://www.ntu.edu.sg/home/cuong/) with
```

Comment:
No need to include me as author. I haven't contributed a line of code ;)

Reply:
Maybe I can be listed as one of the contributors (alongside with Joseph etc.) for "Suggestions on some aspects of the algorithm)

Reply:
The library enjoys many pointers from the TOPP library though :)